src/main/java/com/google/devtools/build/lib/runtime/ConfigExpander.java

```diff
@@ -168,24 +177,26 @@ static void expandConfigOptions(
     }
   }
 
-  private static List<String> getExpansion(
+  private static List<OptionsParser.ArgAndFallbackData> getExpansion(
```

Comment:
Now that `ArgAndFallbackData` is unambiguous, please import it throughout. The outer class name is distracting.